test/lib.test.ts

```diff
@@ -1,6 +1,8 @@
 const path = require("path");
 const tester = require("circom").tester;
 
+export {};
```

Comment:
Just curious, what does it mean?

Reply:
It was new also for me ^^
The tests were giving an error `Cannot redeclare block-scoped variable` and tried different ways to do the imports to avoid having the error but none of them worked. Found this article about the issue https://medium.com/@muravitskiy.mail/cannot-redeclare-block-scoped-variable-varname-how-to-fix-b1c3d9cc8206 and worked fine. This is the reason that is stated in the article:
> It’s a TypeScripts feature. When you writing the code, there needs to be one of the following semantic solutions:
>   - File has been declared as a module and have its own scope.
>   - File has been declared as a script and share the global scope with other scripts.
>
> Unfortunately, there no standard way to implement this division.
> The way that TypeScript decided to solve the problem was to simply state that a module is any file which contains an import or export.
> This way, if your .ts file doesn’t have any sort of top-level import or export statements, then you will see all these issues when global declarations are interfere with each other.